### PR TITLE
feat(hooks): use-draggable support resetPostion

### PR DESCRIPTION
**Pull request by donga-gao**
Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Comment by tolking** [comment]
Thank for your contribution.

Could you briefly explain why this feature is needed?

**Comment by wzc520pyfm** [comment]
The draggabled Dialog can be completely moved out of the window after setting overflow, and it will still be outside the window even if it is opened again. Providing a reset position function would solve this, or could we design a better API?

**Comment by donga-gao** [comment]
> The draggabled Dialog can be completely moved out of the window after setting overflow, and it will still be outside the window even if it is opened again. Providing a reset position function would solve this, or could we design a better API?

you know the scene！

**Comment by donga-gao** [comment]
> The draggabled Dialog can be completely moved out of the window after setting overflow, and it will still be outside the window even if it is opened again. Providing a reset position function would solve this, or could we design a better API?

you know the scene！

**Comment by donga-gao** [comment]
> The draggabled Dialog can be completely moved out of the window after setting overflow, and it will still be outside the window even if it is opened again. Providing a reset position function would solve this, or could we design a better API?

you know the scene！

**Comment by tolking** [comment]
I understand.

You can first improve the Exposes in the document, [refer to](https://github.com/element-plus/element-plus/blob/dev/docs/en-US/component/popover.md#exposes)

**Review by tolking — APPROVED** [review]

File: docs/en-US/component/dialog.md [hunk 1]
```diff
@@ -175,6 +175,12 @@ When using `modal` = false, please make sure that `append-to-body` was set to **
 | open-auto-focus  | triggers after Dialog opens and content focused  | ^[Function]`() => void` |
 | close-auto-focus | triggers after Dialog closed and content focused | ^[Function]`() => void` |
 
+### Exposes
+
+| Name         | Description   | Type                    |
+| ------------ | ------------- | ----------------------- |
+| resetPostion | reset postion | ^[Function]`() => void` |
```
Comment (btea) [inline docs/en-US/component/dialog.md:182]:
```suggestion
| resetPostion ^(2.8.0) | reset postion | ^[Function]`() => void` |
```

File: docs/en-US/component/dialog.md [hunk 2]
```diff
@@ -175,6 +175,12 @@ When using `modal` = false, please make sure that `append-to-body` was set to **
 | open-auto-focus  | triggers after Dialog opens and content focused  | ^[Function]`() => void` |
 | close-auto-focus | triggers after Dialog closed and content focused | ^[Function]`() => void` |
 
+### Exposes
+
+| Name         | Description   | Type                    |
+| ------------ | ------------- | ----------------------- |
+| resetPostion ^(2.8.0) | reset postion | ^[Function]`() => void` |
```
Comment (btea) [inline docs/en-US/component/dialog.md:182]: The version number is added here, which causes the width to change. It should be formatted to be consistent with the header.

**Review by btea — APPROVED** [review]
Thanks!

**Pull request merged**